ooiservices/app/main/user.py

```diff
@@ -36,11 +37,31 @@ def put_user(id):
     if active is not None:
         user_account.active = bool(active)
         changed = True
+        if activating is True:
+            send_activate_email(data["email"])
     if changed:
         db.session.add(user_account)
         db.session.commit()
     return jsonify(**user_account.to_json()), 201
 
+def send_activate_email(to):
+    sender="help@ooi.rutgers.edu"
+    subject="OOI Registration Complete"
+    message=string.join(("From: %s" % sender,
+                         "To: %s" % current_app.config["TOEMAIL"] if current_app.config["ENVIORNMENT"] else to,
+                         "Subject: %s" % subject,
+                         "Your registration to the Ocean Observatories Initiative WebPortal has been completed.",
+                         "",
+                         "",
+                         "Please visit http://casadev.ooi.rutgers.edu to login to the system"),"\r\n")
```

Comment:
Cam, Sorry for this.  Can you make a config variable SERVER_DNS in the config.yml with the defauly value "localhost", then change the code to read:

"Please visit http://%s to login to the system"),"\r\n" %current_app.config["SERVER_DNS"])

If you don't have time, let me know and I'll handle it